addon.gypi

```diff
@@ -52,6 +52,14 @@
         'standalone_static_library': '<(standalone_static_library)'
       }],
 
+      ['library=="shared_library"', {
```

Comment:
I _think_ you want to unset `-fPIE` unconditionally, otherwise it's still going to be problematic with add-ons that also build static libraries.

Reply:
Ok. So we only want to remove the flag if we don't want to compile an executable. Am I right?